scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestApi.java

```diff
@@ -0,0 +1,42 @@
+package org.cloudfoundry.autoscaler.scheduler.rest;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.cloudfoundry.autoscaler.scheduler.rest.model.ApplicationScalingSchedules;
+import org.cloudfoundry.autoscaler.scheduler.service.ScalingScheduleManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author Fujitsu
+ *
+ */
+@RestController
+@RequestMapping(value = "/v2/schedules")
+public class ScheduleRestApi {
+
+	@Autowired
+	ScalingScheduleManager scalingScheduleManager;
+
+	private Log logger = LogFactory.getLog(this.getClass());
+
+	@RequestMapping(method = RequestMethod.PUT)
```

Comment:
Shouldn't this require an appId? If we are using PUT, the id should be provided. If we are using POST, the id can be created. But given we are tying things to an appId, PUT makes more sense but the appId should be in the URL.